test-requirements.txt

```diff
@@ -4,3 +4,5 @@ pytest-datadir
 pytest-runner
 mock>=2.0.0
 flake8
+twine
+wheel
```

Comment:
Do we need wheel here?

Reply:
tox references this file for dependencies, with flake8 also there, I added it preemptively. On my machine however, tox will run the check without these here. I can remove these and as long as GH actions doesn't need them, then they aren't needed at all!

Reply:
Tested this on a test branch, twine is required to pass through GH Actions, but wheel is not. So I will remove wheel.

Used a test branch to avoid branch naming inconsistencies, GH actions requires the name of the branch in order to run the workflow, and my branch and master have two different names.

Reply:
Thanks for testing, sounds good to remove what's not actually necessary.